tests/components/iometer/test_config_flow.py

```diff
@@ -0,0 +1,175 @@
+"""Test the IOmeter config flow."""
+
+from ipaddress import ip_address
+from unittest.mock import AsyncMock
+
+from iometer import IOmeterConnectionError
+
+from homeassistant.components import zeroconf
+from homeassistant.components.iometer.const import DOMAIN
+from homeassistant.config_entries import SOURCE_USER, SOURCE_ZEROCONF
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from tests.common import MockConfigEntry
+
+IP_ADDRESS = "10.0.0.2"
+IOMETER_DEVICE_ID = "658c2b34-2017-45f2-a12b-731235f8bb97"
+
+ZEROCONF_DISCOVERY = zeroconf.ZeroconfServiceInfo(
+    ip_address=ip_address(IP_ADDRESS),
+    ip_addresses=[ip_address(IP_ADDRESS)],
+    hostname="IOmeter-EC63E8.local.",
+    name="IOmeter-EC63E8",
+    port=80,
+    type="_iometer._tcp.",
+    properties={},
+)
+
+
+async def test_user_flow(
+    hass: HomeAssistant,
+    mock_iometer_client: AsyncMock,
+) -> None:
+    """Test full user configuration flow."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={"source": SOURCE_USER},
+    )
+    await hass.async_block_till_done()
+    assert result["type"] is FlowResultType.FORM
+    assert result["step_id"] == "user"
+
+    result = await hass.config_entries.flow.async_configure(
+        result["flow_id"],
+        user_input={CONF_HOST: "10.0.0.2"},
```

Comment:
```suggestion
        user_input={CONF_HOST: IP_ADDRESS},
```
lets use the const in all the other ocorrences too

Reply:
Done